greynoise.json

```diff
@@ -705,23 +732,26 @@
                 {
                     "data_path": "action_result.data.*.link",
                     "data_type": "string",
+                    "contains": [
+                        "url"
+                    ],
                     "example_values": [
-                        "https://viz.greynoise.io/riot/1.2.3.4"
+                        "https://viz.testurl.io/riot/1.2.3.4"
```

Comment:
@dhwanis-crest : why are we removing / changing these sample values?  This seems like an odd pattern?

Reply:
Done